app/controllers/incomeSources/add/AddUKPropertyStartDateController.scala

```diff
@@ -138,4 +138,26 @@ class AddUKPropertyStartDateController @Inject()(val authenticate: Authenticatio
         }
   }
 
+  def change(): Action[AnyContent] =
+    (checkSessionTimeout andThen authenticate andThen retrieveNino
+      andThen retrieveIncomeSources andThen retrieveBtaNavBar).async {
+      implicit user =>
+        //TODO: fix
+        handleRequest(
+          isAgent = false
+        )
+    }
+
+  def changeAgent(): Action[AnyContent] = Authenticated.async {
```

Comment:
lets remove these functions and use show and showAgent instead - otherwise its just dupe code

Reply:
Sure - I think we will need tech debt to address the others..